tests/settings/SettingTest.py

```diff
@@ -44,6 +50,15 @@ def test_url(self):
             uut = Setting("key", "abc")
             url(uut)
 
+    def test_glob_list(self):
+        abspath = os.path.abspath(".")
+        # Need to escape backslashes since we use list conversion
+        self.uut = Setting("key", "., " + abspath.replace("\\", "\\\\"),
+                           origin="test (1)" + os.path.sep + "somefile")
+        self.assertEqual(
+            glob_list(self.uut),
+            [os.path.abspath(os.path.join("test [(]1[)]", ".")), abspath])
```

Comment:
same here, if you use abspath it actually *means* there are non escaped glob components!